browser/ui/views/sidebar/sidebar_add_item_bubble_delegate_view.cc

```diff
@@ -216,3 +244,35 @@ void SidebarAddItemBubbleDelegateView::OnCurrentItemButtonPressed() {
   if (children().size() == 1)
     GetWidget()->CloseWithReason(views::Widget::ClosedReason::kUnspecified);
 }
+
+void SidebarAddItemBubbleDelegateView::OnCreate() {
+  auto* manager = brave_vpn::BraveVPNConnectionManager::GetInstance();
+  brave_vpn::BraveVPNConnectionInfo info;
+  info.connection_name = "Brave VPN";
+  info.hostname = base::UTF16ToUTF8(host_->GetText());
```

Comment:
In we ever wanted to do validation on these params (`hostname` / `username` / `password`), it might be handy to have a set method (ex: `set_hostname` / `set_username` / `set_password`). One example use-case I can see would be if the text needed trimming (removing whitespace / CRLF around beginning or end)

Reply:
Added `SetConnectionInfo()` method.